src/question_tree.rb

```diff
@@ -27,14 +25,17 @@ class QuestionTree < Tree::TreeNode
     def ask_questions
       filtered_each.with_index do |question, index|
         next unless ask_conditional_question?(question)
-        yield(question, index + 1)
+        progress = "(#{index + 1}/#{questions_length})"
```

Comment:
Can we add back the `progress_indicator` method for defining this to this class? This would be a bit clearer I think.

Reply:
I'm not sure, it's an additional method in an already long class. Also the `index` isn't known in advance so it would still need to be an input.

I have however renamed the `progress` variable to `progress_indicator`